yktrace/src/tir.rs

```diff
@@ -5,7 +5,12 @@
 use super::SirTrace;
 use crate::{
     errors::InvalidTraceError,
+<<<<<<< HEAD
```

Comment:
Not sure about this change :P

Reply:
Hah, yeah, cocked that up, but will push the fix together with some other fixes.